client/my-sites/patterns/components/pattern-preview/index.tsx

```diff
@@ -171,7 +198,11 @@ function PatternPreviewFragment( {
 						} }
 						transparent
 					>
-						<Icon height={ 18 } icon={ lock } width={ 18 } /> Get access
+						<Icon height={ 18 } icon={ lock } width={ 18 } />{ ' ' }
+						{ translate( 'Get access', {
+							comment:
+								'Button label for that shows in Pattern Library instead of "Copy" when logged-out users need to sign up',
```

Comment:
I would avoid the mention to the other label, and instead explain what kind of feature the user will get access to:
```suggestion
								'Button label shown when logged-out users need to sign up to be able to use a pattern',
```

Reply:
Good call 👍